src/components/backend-ai-edu-applauncher.ts

```diff
@@ -473,20 +484,32 @@ export default class BackendAiEduApplauncher extends BackendAIPage {
 
     // Launch app.
     if (sessionId) {
-      if (requestedApp.startsWith('jupyter') && !this.detectIE()) {
-        requestedApp = 'jupyterlab';
-      }
       this._openServiceApp(sessionId, requestedApp);
     }
   }
 
   async _openServiceApp(sessionId, appName) {
     this.appLauncher.indicator = await globalThis.lablupIndicator.start();
-    console.log(`launching ${appName} from session ${sessionId} ...`);
+    let requestedApp = appName;
```

Comment:
Not that important thing, but can you unify the naming convention of appName? `eduAppnamePrefix` and `appname` / `eduAppNamePrefix` and `appName`.

Reply:
Changed `eduAppnamePrefix` to `eduAppNamePrefix`, following lowerCamelCase rule.

Reply:
updated webserver in https://github.com/lablup/backend.ai/pull/1738